### ISSUE TITLE
NavmeshUpdaterPlugin is using .after(sync_simple_transforms), which is ambiguous

### ISSUE THREAD

**Issue by 0x0539** [vleue/vleue_navigator#61]
Thanks for the awesome library. I just tried `demo.rs` and it looks great.

I'm trying to incorporate that example into my project, but I'm getting the following error

```
Error when initializing schedule PostUpdate: Tried to order against `SystemTypeSet(fn FunctionSystem<fn(ParamSet<(Query<(Transform, &mut GlobalTransform), (Or<(Changed<Transform>, Added<GlobalTransform>)>, Without<Parent>, Without<Children>)>, Query<(Ref<Transform>, &mut GlobalTransform), (Without<Parent>, Without<Children>)>)>, RemovedComponents<Parent>), sync_simple_transforms>())` in a schedule that has more than one `SystemTypeSet(fn FunctionSystem<fn(ParamSet<(Query<(Transform, &mut GlobalTransform), (Or<(Changed<Transform>, Added<GlobalTransform>)>, Without<Parent>, Without<Children>)>, Query<(Ref<Transform>, &mut GlobalTransform), (Without<Parent>, Without<Children>)>)>, RemovedComponents<Parent>), sync_simple_transforms>())` instance. `SystemTypeSet(fn FunctionSystem<fn(ParamSet<(Query<(Transform, &mut GlobalTransform), (Or<(Changed<Transform>, Added<GlobalTransform>)>, Without<Parent>, Without<Children>)>, Query<(Ref<Transform>, &mut GlobalTransform), (Without<Parent>, Without<Children>)>)>, RemovedComponents<Parent>), sync_simple_transforms>())` is a `SystemTypeSet` and cannot be used for ordering if ambiguous. Use a different set without this restriction.
```

I'm using Bevy 0.14.0 and vleue_navigator 0.10.0 (I can provide more details if helpful).

Here's my guess about what's happening:
1) `sync_simple_transforms` appears twice in the Bevy plugin: 
  -  https://github.com/bevyengine/bevy/blob/f08f07785b1932e1cf6169e604eb3f8e2ac06837/crates/bevy_transform/src/plugins.rs#L39
  - https://github.com/bevyengine/bevy/blob/f08f07785b1932e1cf6169e604eb3f8e2ac06837/crates/bevy_transform/src/plugins.rs#L55
2) Thus this statement is ambiguous: https://github.com/vleue/vleue_navigator/blob/ca915b93cd82c8ea346a7c48a72b6dd8d17fb89e/src/updater.rs#L653

I'm not very confident in my asssment though. Why would the demo work but not my code?

Happy to contribute, but I could use a pointer in the right direction..

Thanks!

**Comment by 0x0539** [comment]
Made a PR that seems to fix this for me: https://github.com/vleue/vleue_navigator/pull/62

**Comment by mockersf** [comment]
fixed by #62, thank you!
